### ISSUE TITLE
Accessibility: Inputs don't meet color contrast ratios

### ISSUE THREAD

**Issue by paceaux** [paceaux/form-baseline#6]
Axe DevTools reports 
> Element has insufficient color contrast of 2.46 (foreground color: #a5a5a5, background color: #ffffff, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1
